script/utils/executeForgeScript.sh

```diff
@@ -3,17 +3,28 @@
 source .env
 
 scriptPath=$1
-shouldVerify=$2
 
-currentGasPrice=$(cast gas-price --rpc-url "$DEPLOYMENT_RPC_URL")
-gasPrice=$(echo "if ($currentGasPrice * 1.25 > 2000000000) ($currentGasPrice * 1.25)/1 else 2000000000" | bc)
+chainId=$(cast chain-id --rpc-url $DEPLOYMENT_RPC_URL)
+gasPrice=$(echo "($(cast gas-price --rpc-url "$DEPLOYMENT_RPC_URL") * 1.25)/1" | bc)
 
-if ! forge script script/$scriptPath --rpc-url "$DEPLOYMENT_RPC_URL" --broadcast --legacy --slow --with-gas-price $gasPrice; then
+if [[ $chainId == "1" ]]; then
+    gasPrice=$(echo "if ($gasPrice > 2000000000) $gasPrice else 2000000000" | bc)
+fi
+
+if [[ $chainId == "1" ]]; then
```

Comment:
Isn't this the same check as immediately above?

Reply:
no idea how this happened. removed